src/modules/organization/InternalGroupDetail.tsx

```diff
@@ -139,14 +142,13 @@ export const InternalGroupDetail: React.VFC = () => {
     variables: { id: internalGroupId },
   })
 
-  if (loading) return <span>Loading</span>
+  if (error) return <span>Loading</span>
 
-  if (!data || error) return <span>Something went wrong</span>
+  if (loading || !data) return <span>Something went wrong</span>
```

Comment:
The return statements are mixed up

Reply:
and I didn't use components for errors/loading. Now using FullPageError and FullContentLoader